web/react/stores/user_store.jsx

```diff
@@ -235,6 +237,14 @@ class UserStoreClass extends EventEmitter {
         BrowserStore.setItem('profiles', ps);
     }
 
+    saveProfiles(profiles) {
+        const currentId = this.getCurrentId();
+        if (currentId in profiles) {
+            delete profiles[currentId];
```

Comment:
Should you just save over this? vs deleting? just nervous that other places may expect current user to be in this list 0/5.